src/main/java/domain/Prize.java

```diff
@@ -0,0 +1,42 @@
+package domain;
+
+import java.util.Arrays;
+
+public enum Prize {
+    FIFTH_PRIZE(3, false, 5_000),
```

Comment:
```suggestion
public enum Prize {

    FIFTH_PRIZE(3, false, 5_000),
```

사소할 수 있지만, 컨벤션의 통일성은 신경써주는 게 좋다고 생각해요.